drake/core/Path.h.in

```diff
@@ -2,7 +2,7 @@
 
 #include <string>
 
-namespace Drake {
+namespace {
```

Comment:
Actually, it was to get the program to link at all without duplicated symbols in every object file which had an include for Path.h.   I'll admit, it was the lazy way, I probably should have changed the code generator to emit Path.cpp instead of Path.h and produce an object file containing getDrakePath.  (an alternate option would be to #define something to @PROJECT_SOURCE_DIR@, but that didn't seem great either).

I'll see how people feel and send out, at a minimum, a fix for the closing comment if we're ok leaving an anon namespace in a header.